srcpkgs/libicns/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'libicns'
+pkgname=libicns
+version=0.8.1
+revision=1
+build_style=gnu-configure
+makedepends="libpng-devel jasper-devel"
+depends="libpng jasper"
```

Comment:
this is automatically detected via shlibs and shouldn't be added explicitley.

Reply:
done